src/scripts/browserDeviceProfile.js

```diff
@@ -536,7 +536,7 @@ export default function (options) {
         }
     }
 
-    if (canPlayAudioFormat('flac')) {
+    if (canPlayAudioFormat('flac') && !browser.tizen) {
```

Comment:
Maybe add a comment that FLAC in video is delayed on Tizen?

Reply:
done!